documentation/changes.md

```diff
@@ -0,0 +1,68 @@
+# Breaking Change Policy
+
+Customers and MVPs have told us many times in emphatic terms that breaking changes are a continuing problem in Azure PowerShell releases.  These changes cause an undue burden on users maintaining scripts to manage their Azure assets, and this causes reluctance in picking up the later versions of Azure PowerShell. It is clear that continuing, unmanaged breaking changes are unnacceptable to customers. 
```

Comment:
@brendanburns  Agreed the extra adjectives are unhelpful.  This is based on user's expectations as expressed - currently there is no stated policy, so no official SLA.